kolibri/content/urls.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
```

Comment:
The URLs and the resources defined here are not RESTful.

You should not use GET parameters to specify the resource id.

See above for my suggestion on how we should namespace the urls for content endpoints by channel_id.